virt/learn_more_about_openshift_virtualization.adoc

```diff
@@ -0,0 +1,108 @@
+include::modules/virt-document-attributes.adoc[]
+[id="learn_more_about_openshift_virtualization"]
+= Learn more about {VirtProductName}
+:context: learn_more_about_openshift_virtualization
+
+toc::[]
+
+Use the following sections to find content to help you learn about and use {VirtProductName}.
+
+[id="cluster-administrator"]
+== Cluster Administrator (Maintenance)
+
+[options="header",cols="4*"]
+|===
+|Learn about {VirtProductName} |Plan for {VirtProductName} deployment |Deploy {VirtProductName}|Additional resources
+
+| xref:../virt/about-virt.adoc#about-virt[Learn about {VirtProductName}]
+| xref:../virt/install/preparing-cluster-for-virt.adoc#preparing-cluster-for-virt[Configuring your cluster for {VirtProductName}]
+| xref:../virt/node_network/virt-updating-node-network-config.adoc#virt-updating-node-network-config[Updating your node network configuration]
+| xref:../virt/logging_events_monitoring/virt-collecting-virt-data.adoc#virt-collecting-virt-data[Getting Support]
+
+| xref:../welcome/learn_more_about_openshift.adoc#learn_more_about_openshift[Learn more about {product-title}]
+| xref:../virt/virtual_machines/virtual_disks/virt-features-for-storage.adoc#virt-features-for-storage[Plan storage for virtual machine disks]
+| xref:../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[Configuring CSI volumes]
+|
+
+| xref:../virt/live_migration/virt-live-migration.adoc#virt-live-migration[Learn about virtual machine live migration]
+|
+| Installing {VirtProductName} using the xref:../virt/install/installing-virt-web.adoc#installing-virt-web[{VirtProductName} console] or xref:../virt/install/installing-virt-cli.adoc#installing-virt-cli[CLI]
+|
+
+| xref:../virt/node_maintenance/virt-about-node-maintenance.adoc#virt-about-node-maintenance[Learn about node maintenance]
+|
+|
+|
+
+|
+|===
+
+[id="virtualizaion-administrator"]
+== Virtualization Administrator (Deployment)
+
+[options="header",cols="4*"]
+|===
+|Learn about {VirtProductName} |Deploy {VirtProductName} |Manage {VirtProductName}|Use {VirtProductName}
```

Comment:
see prior comment about tightening up the table headers.

Reply:
Done